packages/bindgen/spec.yml

```diff
@@ -993,6 +993,7 @@ classes:
       state: SyncUserState
       subscribers_count: count_t
     methods:
+      all_sessions: () -> std::vector<Nullable<SharedSyncSession>>
```

Comment:
I don't think this needs to be `Nullable<>` since we should only get non-null pointers in the vector.

Also, this feels like it could be a property rather than a method. There isn't a hard-and-fast rule, but generally pure getters are turned into properties rather than methods.

Reply:
Ok, i'll follow that guideline and make it into a property and remove `Nullable<>`.  Tests are passing and the SDK code is much simpler 👍🏼